pyvrp/_SubPopulation.pyi

```diff
@@ -49,6 +52,10 @@ class SubPopulation:
         ----------
         individual
             Individual to add to the subpopulation.
+        penalty_manager
+            PenaltyManager to use to compute the cost. Required here
+            since adding an individual may trigger a purge which needs to
+            compute the biased fitness which requires computing the cost.
```

Comment:
Same as in the implementation. This leaks a bit of an implementation detail on _why_ we require it. A user shouldn't need to know this, or care. I think a comment in the implementation is sufficient.